sagemaker-clarify/computer_vision/object_detection/object_detection_clarify.ipynb

```diff
@@ -311,15 +239,18 @@
    "source": [
     "!mkdir -p {TEST_IMAGE_DIR}\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0001.jpg caltech/bear_009_0001.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0002.jpg caltech/bear_009_0002.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/056.dog/056_0023.jpg caltech/056.dog_056_0023.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/056.dog/056_0001.jpg caltech/056.dog_056_0001.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0003.jpg caltech/064.elephant-101_064_0003.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0004.jpg caltech/064.elephant-101_064_0004.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0006.jpg caltech/064.elephant-101_064_0006.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/250.zebra/250_0001.jpg caltech/250.zebra_250_0001.jpg\n",
-    "!aws s3 cp s3://sagemaker-sample-files/datasets/image/caltech-256/256_ObjectCategories/250.zebra/250_0002.jpg caltech/250.zebra_250_0002.jpg"
+    "\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0001.jpg > ./caltech/09_0001.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0002.jpg > ./caltech/bear_009_0002.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0002.jpg > ./caltech/bear_009_0002.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/009.bear/009_0002.jpg > ./caltech/bear_009_0002.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/056.dog/056_0023.jpg > ./caltech/056.dog_056_0023.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/056.dog/056_0001.jpg > ./caltech/056.dog_056_0001.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0003.jpg > ./caltech/064.elephant-101_064_0003.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0004.jpg > ./caltech/064.elephant-101_064_0004.jpg\n",
+    "!curl  https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/caltech-256/256_ObjectCategories/064.elephant-101/064_0006.jpg > ./caltech/064.elephant-101_064_0006.jpg\n",
```

Comment:
Can you pull these image names into an array and loop over them to make this a little cleaner? I know this is from the original code but this is good opportunity for a cleanup ;)

Reply:
updated.